src/backEnd/controllers/add.js

```diff
@@ -24,16 +25,14 @@ exports.post = (req, res) => {
         price,
         image_url,
         userId,
-        serviceId: 2,
+        serviceId: 1,
```

Comment:
You have never adding static data, here 'serviceId' value should return from front-end